client/components/spinner-button/style.scss

```diff
@@ -0,0 +1,4 @@
+.spinner-button__spinner {
+	float: right;
+	padding: 8px;
+}
```

Comment:
Nice, installed the plugin and fixed the newline in 8032f95b51c1cd841af0768724d420f1c91da8f3